src/schema/schema.graphql

```diff
@@ -1125,6 +1125,43 @@ type Folder {
 	unreadDescendent: Boolean
 }
 
+type DlAttrs {
+	commonName: String
+	displayName: String
+	lastName: String
+}
+
+type DlGroupMember {
+	name: String
+	_attrs: DlAttrs
+}
+
+type DlGroupMembers {
+	groupMember: [DlGroupMember]
+}
+
+type HabGroup {
+	name: String
+	id: String
+	parentHabGroupId: ID
+	seniorityIndex: Int
```

Comment:
can this contain value which is larger then 2lac, if yes then convert it to Float

Reply:
seniorityIndex would never be that larger, so keeping it to Int only. I did use Int by following Zimbra API Docs.